core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2104,4 +2104,17 @@ private CarbonCommonConstants() {
    */
   public static final String CARBON_QUERY_DATAMAP_BLOOM_CACHE_SIZE_DEFAULT_VAL = "512";
 
-}
+  public static final String CARBON_INDEX_SERVER_POLICY = "carbon.index.server.policy";
```

Comment:
please write description for this config in comment, like in what scenario this config is used, what valid values are, etc